MQ-CNN algo: error message on kernel_size_seq is confusing

## Description
Having ones in the array param `kernel_size_seq` of `gluonts.model.seq2seq.MQCNNEstimator` raises an error indicating that the values in the array **should all be positive**. Quickly looking into the source code, there seems to be a mismatch between the check performed in the assertion and the error message subsequently raised (https://ts.gluon.ai/stable/_modules/gluonts/model/seq2seq/_mq_dnn_estimator.html#MQCNNEstimator):
<img width="410" alt="image" src="https://user-images.githubusercontent.com/22995764/191609310-d499dff8-8c74-4117-bb9e-7aa013531a3b.png">

## To Reproduce
Can be seen with this simple/dummy code:
```python
from gluonts.model.seq2seq import MQCNNEstimator
MQCNNEstimator(freq=10, prediction_length=10, kernel_size_seq=[1, 4, 8])
```

## Error message or code output
`AssertionError: Elements of `kernel_size_seq` should be > 0`

## Environment
- Python version: 3.6
- GluonTS version: 0.10.4
- MXNet version: 1.7.0.post2

--- a/src/gluonts/mx/block/cnn.py
+++ b/src/gluonts/mx/block/cnn.py
@@ -97,7 +97,8 @@ class CausalConv1D(gluon.HybridBlock):
         """
         ct = self.conv1d(data)
         if self.kernel_size > 0:
-            ct = F.slice_axis(ct, axis=2, begin=0, end=-self.padding)
+            end_ = -self.padding if self.padding != 0 else None
+            ct = F.slice_axis(ct, axis=2, begin=0, end=end_)
         return ct
 
 

--- a/src/gluonts/mx/model/seq2seq/_mq_dnn_estimator.py
+++ b/src/gluonts/mx/model/seq2seq/_mq_dnn_estimator.py
@@ -182,9 +182,8 @@ class MQCNNEstimator(ForkingSeq2SeqEstimator):
         assert dilation_seq is None or all(
             d > 0 for d in dilation_seq
         ), "Elements of `dilation_seq` should be > 0"
-        # TODO: add support for kernel size=1
         assert kernel_size_seq is None or all(
-            d > 1 for d in kernel_size_seq
+            d > 0 for d in kernel_size_seq
         ), "Elements of `kernel_size_seq` should be > 0"
         assert quantiles is None or all(
             0 <= d <= 1 for d in quantiles

